site/layouts/_default/examples.html

```diff
@@ -95,6 +95,7 @@
     {{- end }}
   </head>
   <body{{ with .Page.Params.body_class }} class="{{ . }}"{{ end }}>
+    {{ if ne .Page.Params.theme_selector false }}
     <svg xmlns="http://www.w3.org/2000/svg" style="display: none;">
```

Comment:
Maybe we could keep this svg in each example since we introduce the theme selector anyway ? If yes, we could remove newly added svgs from the examples.

Reply:
Done via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2427/commits/14de9ca1dd40a0f6372bcc6e5679fb683bb677c4